docs/02-Install.md

```diff
@@ -105,7 +65,51 @@ In the LORIS front-end, under the Admin menu, go to the `Config` module. Under
 
 Click 'Submit' at the end of the Configuration page to save any changes.
 
-#### 2.4.5 FAQ
+#### 2.3.5 FAQ
 
 For common errors and frequently asked questions, please refer to the [Appendix
-  A](AppendixA-FAQ.md).
\ No newline at end of file
+  A](AppendixA-FAQ.md).
+
+
+## 2.4 Pipeline flow
+
+The pipeline was initially designed for **raw DICOM MRI data**, collected by a
+  longitudinally-organized multi-site study with a defined imaging acquisition
+  protocol. With modifications and further customization, it can handle any
+  modality of imaging data.
+
+Typically, images insertion into LORIS is performed via the following steps:
+
+1. Upload of a compressed set of the DICOM study via the **Imaging Uploader**
+     module (or transfer the compressed DICOM studies directly on the server
+     hosting LORIS).
+2. DICOM insertion into the `tarchive` tables in order to be able to see the
+     information stored in the DICOMs via the **DICOM Archive** module.
+3. Conversion of the DICOMs into MINC files for protocol validation and
+     insertion into the LORIS database. By default it will also create NIfTI
+     images from the MINC files and insert them into LORIS. (See the
+     [Configuration](#2.2-configuration) section above if you want to disable
+     this option.) One of two possible actions will follow depending on the
+     study-defined protocol:
+     
+    a. If a scan matches one of the protocol defined in the `mri_protocol`
```

Comment:
here, there is a subtle detail omitted; and that is the extra_file_checks() using the other (optional) protocol table. Do you think this should be mentioned here?